tests/foreman/ui/test_reporttemplates.py

```diff
@@ -385,12 +386,12 @@ def test_positive_autocomplete(session):
 
 @pytest.mark.tier2
 def test_positive_schedule_generation_and_get_mail(
-    session, module_org, module_location, default_sat
+    session, module_manifest_org, module_location, default_sat
 ):
     """Schedule generating a report. Request the result be sent via e-mail.
 
     :id: cd19b90d-836f-4efd-c3bc-d5e09a909a67
-    :setup: User with reporting access rights, some Host
+    :setup: User with reporting access rights, some Host, org with imported manifest
```

Comment:
The caps here are used to indicate entity names.
```suggestion
    :setup: User with reporting access rights, some Host, Org with imported manifest
```